src/components/FlowChart/FlowSelection.jsx

```diff
@@ -1,56 +1,118 @@
-import React from "react";
+import React, { useState } from "react";
 import { Dropdown } from "react-bootstrap";
 
+function SelectionIndicator({ selectedId, itemId }) {
+  return selectedId === itemId ? (
+    <i className="material-icons text-primary">radio_button_checked</i>
+  ) : (
+    <i className="material-icons">radio_button_unchecked</i>
+  );
+}
+
+function ConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons ml-2 text-success">wifi</i>
+  ) : (
+    <i className="material-icons ml-2 text-warning">wifi_off</i>
+  );
+}
+
+function TitleConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons text-white ml-1 mr-2">wifi</i>
+  ) : (
+    <i className="material-icons text-warning ml-1 mr-2">wifi_off</i>
+  );
+}
```

Comment:
```jsx
function TitleConnectionIndicator({ isConnected }) {
  return  <i className="material-icons text-warning ml-1 mr-2">{isConnected? 'wifi': 'wifi_off'}</i>
}
```

Less duplication. `isConnected` makes it easier to see, that it should be a boolean.